index.html

```diff
@@ -1997,6 +1997,40 @@ <h5>Get checksum for video/audio stream</h5>
     </div>
     <!-- ends Get checksum for video/audio stream -->
 
+    <!-- Get checksum for all video/audio streams -->
+    <label class="recipe" for="get_streamhash">Get individual checksums for all video/audio streams ("Streamhash")</label>
+    <input type="checkbox" id="get_streamhash">
+    <div class="hiding">
+      <h5>Get individual checksums for all video/audio streams ("Streamhash")</h5>
+      <p><code>ffmpeg -i <em>input_file</em> -map 0 -f streamhash -hash md5 - -v quiet</code></p>
+      <p>The outcome is very similar to that of "-f hash", except you get one hash
+      per-stream, instead of one (summary) hash.  Another benefit is that you
+      don't have to know which streams, or how many to expect in the source
+      file. This is very handy for hashing mixed born-digital material.</p>
+      <p>This script will perform a fixity check on all audio and video streams in
+      the file and return one hashcode for each one.  This is useful for e.g.
+      be able to change to container/codec format later on and validate it
+      matches the original source.</p>
```

Comment:
'so that it matches'

Reply:
Huh, the part I would change is `e.g. be able` -> `e.g. being able`